code/game/objects/effects/spawners/depot_spawners.dm

```diff
@@ -164,16 +169,18 @@
 	// Combat orientated items that could give the player an advantage if an antag messes with them.
 	loot = list(
 		/obj/item/autosurgeon/organ/syndicate/oneuse/razorwire,
+		/obj/item/autosurgeon/organ/syndicate/oneuse/sensory_enhancer,
+		/obj/item/bio_chip_implanter/proto_adrenalin,
 		/obj/item/chameleon,
-		/obj/item/clothing/gloves/fingerless/rapid,
 		/obj/item/CQC_manual,
 		/obj/item/gun/medbeam,
+		/obj/item/gun/projectile/automatic/sniper_rifle/toy,
 		/obj/item/melee/energy/sword/saber,
+		/obj/item/mod/control/pre_equipped/traitor_elite,
```

Comment:
The addition of the elite suit was not mentioned in the PR description. Please add it when available!

Code looks good.

Reply:
elite modsuit already was there, i just moved it around the list

Reply:
Oops, I missed it all the way at the bottom!